aioftp/client.py

```diff
@@ -547,13 +697,38 @@ def list(self, path="", *, recursive=False):
 
             >>> stats = await client.list()
         """
-        class AsyncClientLister(AsyncListerMixin):
+        class AsyncListerClient(AsyncListerMixin):
 
             async def _new_stream(cls, local_path):
 
                 cls.path = local_path
-                command = str.strip("MLSD " + str(cls.path))
-                return await self.get_stream(command, "1xx")
+                str_path = " " + str(cls.path)
+                if self.list_compatibility_level > 0:
+
+                    cls._anext = cls._anext_MSLD_LIST
+
+                    if self.list_compatibility_level == 2:
+
+                        cls.process_line = self.parse_mlsx_line
```

Comment:
More obvious name for `process_line` is `parse_line` this will echo with `parse_mlsx_line` and `parse_list_line`.